### DISCUSSION TITLE
What is the difference between == and === operators?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#346] [Q&A] [answered]
In short

**Comment by d4tdev** [comment, marked as answer]
The equality operator in javascript is used to compare if two values are equal. The comparison is made by == and === operators in javascript. The main difference between the == and === operator in javascript is that the == operator does the type conversion of the operands before comparison, whereas the === operator compares the values as well as the data types of the operands

**Comment by Preetiraj3697** [comment]
Double Equals ( == ) checks for value equality only. It inherently does type coercion. This means that before checking the values, it converts the types of the variables to match each other. On the other hand, Triple Equals ( === ) does not perform type coercion.
